### PR TITLE
EC 315 create route and view: /starting block/numberOfBlocks

### PR DESCRIPTION
**Pull request by GusevTimofey**

### CODE REVIEW

File: app/controllers/HistoryController.scala [hunk 1]
```diff
@@ -15,7 +16,7 @@ class HistoryController @Inject()(cc: ControllerComponents, historyService: Hist
   def getHeader(id: String): Action[AnyContent] = Action.async {
     historyService
       .getHeader(id)
-      .map(header => Ok(header.asJson))
+      .map(header => Ok(getHeaderView(header.get)))
```
Comment (jhnsmth) [inline app/controllers/HistoryController.scala]: 🙀 get 🙀 get 🙀 get 🙀 get 🙀 get 🙀

File: app/controllers/HistoryController.scala [hunk 2]
```diff
@@ -33,7 +34,7 @@ class HistoryController @Inject()(cc: ControllerComponents, historyService: Hist
   def getBestHeaderAtHeight(height: Int): Action[AnyContent] = Action.async {
     historyService
       .getBestHeaderAtHeight(height)
-      .map(header => Ok(header.asJson))
+      .map(header => Ok(getHeaderView(header.get)))
```
Comment (jhnsmth) [inline app/controllers/HistoryController.scala]: 🙀 get 🙀 get 🙀 get 🙀 get 🙀 get 🙀

File: app/controllers/HistoryController.scala [hunk 3]
```diff
@@ -2,23 +2,31 @@ package controllers
 
 import io.circe.syntax._
 import javax.inject.{Inject, Singleton}
+import models.Header
 import play.api.libs.circe.Circe
 import play.api.mvc._
 import services.HistoryService
+import views.html.{getHeader,getHeaderList}
+
```
Comment (jhnsmth) [inline app/controllers/HistoryController.scala]: empty line

File: app/models/database/HeadersQueryRepository.scala [hunk 4]
```diff
@@ -20,5 +20,8 @@ object HeadersQueryRepository {
 
   def findByHeightRangeQuery(from:Int, to: Int): ConnectionIO[List[Header]] =
     sql"SELECT * FROM headers WHERE height BETWEEN $from AND $to".query[Header].to[List]
+
+  def findByCountQuery(from:Int, count: Int): ConnectionIO[List[Header]] =
+    sql"SELECT * FROM headers WHERE height BETWEEN $from AND ($from + $count - 1)".query[Header].to[List]
```
Comment (GRYE) [inline app/models/database/HeadersQueryRepository.scala]: I think query should be smth like `SELECT * FROM headers WHERE height >=  $from LIMIT $count`

**Pull request merged**